app.py

```diff
@@ -37,7 +37,24 @@ def create_story():
             "families": {
                 story["familyId"]: True
             }
-        })
+        })        
         response = app.response_class(
             response=json.dumps(new_story.get()), status=201, mimetype="application/json")
         return response
+
+
+@app.route('/users', methods=['POST'])
+@cross_origin()
+def create_user():
+    if request.method == "POST":
+        request_data = request.get_json()
+        users = db.reference("users/" + request_data["userId"])
+        res = users.set({
+           	"email": request_data["email"], 
+	        "fullName": request_data["fullName"],
+	        "displayName": request_data["displayName"],
+	        "familyName": request_data["familyName"],
+            "fid": str(uuid.uuid4()),
+        })
+        return request_data, 201
```

Comment:
does this work/does this match what we are expecting in the front end or do you intend to change that? we definitely need the family id back.